pages/linux/aspell.md

```diff
@@ -0,0 +1,23 @@
+# aspell
+
+> Interactive spell checker.
+
+- Spell check a single file:
+
+`aspell check file`
+
+- List mispelled words from standard input:
+
+`cat file | aspell list`
+
+- Show available dictionary files:
+
+`aspell dicts`
+
+- Run aspell with different language (takes two letter ISO 639 language code):
+
+`aspell --lang=lang-code`
+
+- List mispelled words from standard input and ignore words from personal word list:
+
+`cat file |aspell --personal=personal-word-list.pws --dont-suggest list`
```

Comment:
Same here with `personal-word-list.pws`.

Also, can the user change the `list` parameter here after `--dont-suggest`? If so, perhaps that could be surrounded in `{{` and `}}` too. If it's the path to a file that contains the list, maybe we could have `{{path/to/list}}`?

Reply:
done